installing/installing_oci/installing-oracle-cloud-infra-overview.adoc

```diff
@@ -0,0 +1,61 @@
+//Useful links
+// https://docs.google.com/document/d/1yG4L0boO3z1_jZWiDpK_m9eVihqS1RYNGJ7sB5C8DTI/edit#heading=h.5q9j11tnaspw[Content plan]
+// Slack: https://app.slack.com/client/T027F3GAJ/C04T569EL1Z/thread/C04T569EL1Z-1692001689.865169
+// https://www.oracle.com/cloud/compute/vmware/
+// https://cloud.vmware.com/providers/cloud-providers/oracle
+// https://www.oracle.com/cloud/distributed-cloud/
+// https://www.redhat.com/en/about/press-releases/red-hat-and-oracle-expand-collaboration-bring-red-hat-openshift-oracle-cloud-infrastructure
+// https://cloud.redhat.com/blog/running-your-red-hat-openshift-container-platform-4.12-on-oracle-cloud-vmware-solution-ocvs
+// https://vmc.techzone.vmware.com/resource/red-hat-openshift-container-platform-412-oracle-cloud-vmware-solution#executive-summary
+// https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm
+// https://catalog.redhat.com/cloud/detail/216977
+// Prcoedure similar to https://docs.openshift.com/container-platform/4.12/installing/installing_platform_agnostic/installing-platform-agnostic.html
+// https://github.com/openshift/infrastructure-provider-onboarding-guide/pull/16/files
+
+:_content-type: ASSEMBLY
+[id="installing-oracle-cloud-infra-overview"]
+= Installing a cluster on OCI overview
+include::_attributes/common-attributes.adoc[]
+:context: installing-oracle-cloud-infra-overview
+:platform: OCI
+
+toc::[]
+
+//:oracle: Oracle&#174;
+//:oci: Oracle&#174; Cloud Infrastructure (OCI)
+//:ocvs: Oracle&#174; Cloud VMware Solution (OCVS)
+//:oci-short: OCI
+//:ocvs-short: OCVS
+
+You can install an {product-title} cluster on {oci}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments. Both Red Hat and {oracle} certify, test, and support running {oci-short} and {ocvs} workloads in an {product-title} cluster on {oci}.
+
+{oci-short} provides services that can meet your regulatory compliance, performance, and cost-effectiveness needs. Additionally, {oci-short} provides an {ocvs} service where you can move VMware workloads to {oci-short} with minimal application re-architecture.
+
+By running your {product-title} cluster on {oci}, you can access the following capabilities:
+
+* Compute flexible shapes, where you can customize the number of {oracle} CPUs (OCPUs) and memory resources for your virtual machine (VM). With access to this capability, a cluster's workload can perform operations in a resource-balanced environment. 
```

Comment:
we need to let customers also know to check to use the [RHEL certified Oracle Cloud Infrastructure shapes](https://catalog.redhat.com/cloud/detail/216977) (instance types).

CC @makentenza @racedo

Reply:
Thanks. Updated!